core.php

```diff
@@ -274,6 +274,7 @@ function __autoload( $p_class ) {
 
 # Set HTTP response headers
 require_api( 'http_api.php' );
+
```

Comment:
It seems that you reverted the code changes here and left with some blanks. Can you revert to original core.php before these changes to remove the noise.